.github/CONTRIBUTING.md

```diff
@@ -23,6 +23,12 @@ yarn test-unit
 
 Make sure all the tests pass before making changes.
 
+### Running Vercel CLI Changes
+
+You can use `yarn dev <cli-commands...>` to invoke Vercel CLI with local changes.
```

Comment:
I think this needs `cd ./packages/cli` first

Reply:
Agreed. Check out this follow-up PR: https://github.com/vercel/vercel/pull/8964